src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -18081,8 +18081,8 @@ public String removeMilkModifier(FluidModifier fluidModifier) {
 	// Misc:
 	public boolean hasBreastsCrotch() {
 		if(Main.getProperties().udders==0
-				|| (this.getLegConfiguration()==LegConfiguration.BIPEDAL && Main.getProperties().udders==1)
-				|| (this.getLegConfiguration()==LegConfiguration.BIPEDAL && this.getRaceStage()!=RaceStage.GREATER)) {
```

Comment:
These lines were here to make sure that players with crotch-boobs turned off in the options would never see their descriptions. Surely by commenting this out, it disables that behaviour, thus making it so all players see crotch-boobs regardless of settings?

Reply:
@Innoxia True, but something is nonetheless wrong with it, because - as written - there is no setting that allows crotch-boobs to be visible on any character who is not a Greater Furry. This means that crotch-boobs spuriously disappear off of both Lesser Furries and Hybrids.